Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,17 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes/archive/v2.2.1.tar.gz"
+  sha256 "b0c537f078e8ba2657da8f66c9ac5b0ed31d558c55bd0fff3b7898791dee1a1e"
+
+  depends_on "tmux" => :recommended
+
+  def install
+    bin.install Dir["bin/*"]
+    man1.install Dir["man/*.1"]
+  end
+
+  test do
+    system "#{bin}/xpanes", "-V"
```

Comment:
Did you come up with any ideas on a functionality test in the end here?

Reply:
Tests were added (35d766e30).
Testing for checking version number format and validity of combination of options.
I have tried to implement functional tests which create tmux session. But it didn't work because it seems that creating new terminal is not allowed during the `brew test`.